### PR TITLE
edited README for correct production and storyboard links

### LINKED ISSUE #2: Set up -docs and -docs-qa websites to host Storybook.
Set up -docs and -docs-qa websites to host Storybook.

## General Acceptance criteria

- [x] A personal access token with scope `repo` has been added
      under `Secrets`->`Actions` with key `DOCS_TOKEN`.  

## Acceptance Criteria for `team02-yourTeam-docs-qa` 

- [x] A public repo with the name `team02-yourTeam-docs-qa` exists.
- [x] There is a `README.md` file and a `docs/.keep` file on the `main` 
      branch of the `team02-yourTeam-docs-qa` repo.
- [x] GitHub pages is enabled on the `team02-yourTeam-docs-qa` repo.
- [x] The README.md has a correct link to the documentation for the QA site (done via a pull request; see "Details-1" below)

## Acceptance Criteria for `team02-yourTeam-docs` 

- [x] A public repo with the name `team02-yourTeam-docs` exists.
- [x] There is a `README.md` file and a `docs/.keep` file on the `main` 
      branch of the `team02-yourTeam-docs` repo.
- [x] GitHub pages is enabled on the `team02-yourTeam-docs` repo.
- [x] The README.md has a correct link to the documentation for the QA site (this should happen when you merge the PR from Details-1; see "Details-2" below)

## Details-1: Pull request to trigger initial docs setup

1. Make a new branch `xy-setup-docs` where `xy` are your intials.
   If working as a pair, use `wx-yz-setup-docs`.

   ```text
   git checkout main
   git pull origin main
   git checkout -b xy-setup-docs
   ```

2. Working on this branch, edit the `README.md` to make the links near the top to the GitHub pages sites for documentation point to the correct spot (e.g. edit this:

   ```md
   Storybook is here:
   * Production: <https://ucsb-cs156-w22.github.io/starter-team02-docs/>
   * QA:  <https://ucsb-cs156-w22.github.io/starter-team02-docs-qa/>
   ```

   to say this (substituting your team in for `w22-7pm-3`)

   ```md
   Storybook is here:
   * Production: <https://ucsb-cs156-w22.github.io/team02-w22-7pm-3-docs/>
   * QA:  <https://ucsb-cs156-w22.github.io/team02-w22-7pm-3docs-qa/>
   ```

3. Do a pull request to the main branch.  This should trigger the GitHub
   Action workflow that publishes the documentation to the QA site.

   If this step fails, the usual cause is some problem related to  the personal access token called `DOCS_TOKEN`; either it has:
   * the wrong scope (it should be `repo`)
   * the wrong name (it should be `DOCS_TOKEN`)
   * or was uploaded to the wrong place.

   If it doesn't work, try fixing the token and then re-running the job.

4. Check the link for the QA site and ensure that the 
   documentation shows up.

## Details-2: Pull request to trigger initial docs setup

Once the qa-site is working properly, you should merge the pull request
for `xy-setup-docs` branch that you created.

This should trigger another GitHub actions run that will set up the 
repo for the production documentation.

### PR DESCRIPTION
**Pull request by tai-jui**
# Overview

This PR needs to be approved  to trigger the GitHub
Action workflow that publishes the documentation to the QA site.

# Issues Addressed

If this PR closes or deals with issues, mention them and tag the issue number with the `#` syntax.

# Details

Where applicable, include "before" and "after" screenshots or animated GIFs.   The tool [LiceCAP](https://www.cockos.com/licecap/) or one of [these similar tools](https://www.nextofwindows.com/5-free-tools-to-screen-capture-to-gif-on-windows) 
can be used to capture screenshots/animated GIFs.

## Reminders (delete this section before merging or as code is merged)
- [ ] Explicitly link to this pull request each issue that it closes
- [ ] Assign one and ideally two reviewers to review this pull request. 
- [ ] Reviewers should not just approve with "Looks good!" Instead, reviewers should carefully go through the code and use GitHub's reviewer interface to ask questions, make comments, and suggest changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-7pm-2/branch/main/graph/badge.svg?token=rZT26BG1Dj)](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-7pm-2)
 
 Storybook is here:
-* Production: <https://ucsb-cs156-w22.github.io/starter-team02-docs/>
-* QA:  <https://ucsb-cs156-w22.github.io/starter-team02-docs-qa/>
+
+- Production: <https://ucsb-cs156-w22.github.io/team02-w22-7pm-2-docs/>
+- QA: <https://ucsb-cs156-w22.github.io/team02-w22-7pm-2docs-qa/>
```
Comment (AHuangHe) [inline README.md]: A small mistake here (just missing a dash after 7pm-2)
should be https://github.com/ucsb-cs156-w22/team02-w22-7pm-2-docs-qa for the QA

**Review by AHuangHe — CHANGES_REQUESTED** [review]
There was a small mistake
Also, there's a conflict with the previous merge

**Review by edisonnchen — APPROVED** [review]
LGTM

**Review by AHuangHe — APPROVED** [review]
LGTM tho I am not sure why codecov is complaining

**Pull request merged**